src/Features/ERDDAP/Platform/Observations/CurrentConditions/error_card.tsx

```diff
@@ -0,0 +1,26 @@
+import React from "react"
```

Comment:
Codacy found an issue: ['React' was used before it was defined.](https://app.codacy.com/manual/gmri/Neracoos-1-Buoy-App/pullRequest?prid=6166252)